blaze-server/src/main/scala/org/http4s/server/blaze/Http1ServerStage.scala

```diff
@@ -87,7 +87,7 @@ private[blaze] class Http1ServerStage[F[_]](
 
   private val handleReqRead: Try[ByteBuffer] => Unit = {
     case Success(buff) => reqLoopCallback(buff)
-    case Failure(Cmd.EOF) => stageShutdown()
+    case Failure(Cmd.EOF) => closeConnection()
```

Comment:
Idiom question for blaze-0.14: should we be shutting down here, or waiting for a `Disconnected` message to find its way back to us?